gadgets/trace_tcpconnect/README.mdx

```diff
@@ -28,10 +30,134 @@ Running the gadget:
     </TabItem>
 </Tabs>
 
-## Flags
+## Guide
 
-No flags.
```

Comment:
Note that by removing the flags sections, it will not be readded automatically when new flags are added in the gadget because the sed command would not find it:
https://github.com/inspektor-gadget/inspektor-gadget/blob/main/gadgets/Makefile#L79

In my opinion, it is fine to keep the flags section. Or the Makefile could be updated.